test/gleam/bit_array_test.gleam

```diff
@@ -308,3 +308,30 @@ pub fn inspect_partial_bytes_test() {
   bit_array.inspect(<<5:3, 11:4, 1:2>>)
   |> should.equal("<<182, 1:size(1)>>")
 }
+
+pub fn find_found_test() {
+  let needle = <<"bar":utf8>>
+  <<"foobar":utf8>>
```

Comment:
Could you swap "foobar" for something else? It's a rather depressing WW2 reference and I prefer to keep it light-hearted. Thanks

Reply:
I know zero erlang, I don't know or can seem to find the erlang function for it. I implemented the JS one though.
I didn't know that was a ww2 reference, i changed the tests when changing them for split_once.